releasenotes/notes/cws-7-49-5a11d31bba5f9051.yaml

```diff
@@ -0,0 +1,22 @@
+---
+features:
+  - |
+    CWS: Detect patterns in processes and files paths to improve
+    accuracy of anomaly detections.
+enhancements:
+  - |
+    CWS: Support fentry/fexit eBPF probes which provide lower overhead than
+    kprobe/kretprobes (currently disabled by default and supported only
+    on Linux kernel 5.10 and later).
+  - |
+    CWS: Improved username resolution in containers and handle their creation
+    and deletion at runtime.
+  - CWS: Apply policy rules on processes already present at startup.
```

Comment:
I'm wondering if this is not going to be interpreted as a `key: value` instead of a string literal, maybe you cnan use the `- |` trick everywhere ?

Reply:
Yes, you're probably right, thanks